.gitignore

```diff
@@ -10,3 +10,5 @@
 /php-cs-fixer.phar.asc
 /phpunit.xml
 /vendor/
+.env
```

Comment:
Why these 2? Are they added in this PR?

Reply:
`.env` was added because at the beginning I used env vars in Compose dist file to allow customising XDebug config. But then I realised it's already possible since dist is copied to actual file and can be customised. Anyway, I left it here because it still can be used in Compose file (Docker reads it automatically).

`xdebug.log` is added to `docker/php-8.2/xdebug.ini` and we should ignore it.

Reply:
Can you then align it to the other entries by adding leading `/` and keep them sorted?

Reply:
Done for `.env`, `xdebug.log` needs to be without leading `/` because it's placed in subdirectory and we want to ignore logs for every PHP version.